src/room-creation/RoomCreationModal.svelte

```diff
@@ -21,7 +21,7 @@
             method: 'POST',
             body: JSON.stringify(roomForm)
         });
-        roomForm = await res.json();
+        roomForm = <Room>(await res.json());
```

Comment:
To open the general discussion: I prefer doing type assertions this way:

```suggestion
roomForm = (await res.json()) as Room;
```

I think they might be semantically equal, just a preference. What do you think?

Reply:
It's prettier, updated :)